Mathlib/Algebra/Category/BoolRing.lean

```diff
@@ -97,23 +97,21 @@ end BoolRing
 -- Porting note: Added. somehow it does not find this instance.
 instance {X : BoolAlg} :
     BooleanAlgebra ↑(BddDistLat.toBddLat (X.toBddDistLat)).toLat :=
-  BoolAlg.instBooleanAlgebra _
+  BoolAlg.str _
 
 instance {R : Type u} [BooleanRing R] :
     BooleanRing (BoolAlg.of (AsBoolAlg ↑R)).toBddDistLat.toBddLat.toLat :=
   inferInstanceAs <| BooleanRing R
 
 @[simps]
 instance BoolRing.hasForgetToBoolAlg : HasForget₂ BoolRing BoolAlg where
-  forget₂ :=
-    { obj := fun X ↦ BoolAlg.of (AsBoolAlg X)
-      map := fun {R S} f ↦ RingHom.asBoolAlg f.hom }
+  forget₂.obj X := BoolAlg.of (AsBoolAlg X)
```

Comment:
There are plenty of potential uses of anonymous dot notation. Would you like to act on them?
```suggestion
  forget₂.obj X := .of (AsBoolAlg X)
```

Reply:
I went through the diff and turned every `FooCat.of` into `.of` where it worked.